src/models/teamModel.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const teamSchema = new Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  users: [
+    {
+      type: Schema.Types.ObjectId,
+      ref: 'User',
+    },
+  ],
```

Comment:
The user's role is missing in order to know which right he has in relation to the team
```suggestion
  members: [{
   user: {
      type: Schema.Types.ObjectId,
      ref: 'User',
      require: true,
    },
   role: {
      type: String,
      default: 'member',
    } 
  }],
```

Reply:
I agree with this change, because an user can have a role in all app but have right in only his team